.spelling

```diff
@@ -385,6 +385,8 @@ RPCs
 sa
 sayin
 schemas
+scope1
```

Comment:
I don't think scope1 and scope2 should be in the dictionary. Instead, put backticks around the use of these strings in the text.

Reply:
Even better, separate the term from the number. "scope 1" and "scope 2" cause no misspelling.

Reply:
Done: removed these two words from the dictionary.